controllers/class-wc-ebanx-api-controller.php

```diff
@@ -28,6 +28,26 @@ public function dashboard_check() {
 		));
 	}
 
+	/**
+	 * Responds that the plugin logs
+	 *
+	 * @return void
+	 */
+	public function retrieve_logs() {
+		header('Content-Type: application/json');
+
+		if(empty($_GET['integration_key']) || $_GET['integration_key'] !== $this->get_integration_key()) {
```

Comment:
We need to fix at least this kind of error.
`a('b', 'c');`
should be
`a( 'b', 'c' );`